android/java/org/chromium/chrome/browser/crypto_wallet/adapters/WalletOnboardingPagerAdapter.java

```diff
@@ -0,0 +1,228 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
2024

Reply:
I don't know why it's not indicated by GitHub UI (it usually does it), but this file has been renamed from `CryptoWalletOnboardingPagerAdapter` I haven't implemented it from scratch, shall I update the copyright year anyway?